code/_core/obj/structure/interactive/door/airlock/_airlock.dm

```diff
@@ -317,6 +320,18 @@ obj/structure/interactive/door/airlock/close(var/mob/caller,var/lock = FALSE,var
 		if(!L.dead)
 			open(L)
 
+obj/structure/interactive/door/clicked_on_by_object(var/mob/caller,var/atom/object,location,control,params)
+
+	INTERACT_CHECK
+	INTERACT_CHECK_OBJECT
+	INTERACT_DELAY(1)
+
+	if(is_item(object))
+		var/obj/item/objI = object
+		if(objI.flags_tool & FLAG_TOOL_CROWBAR && door_state == DOOR_STATE_CLOSED)
+			open(caller, FALSE, TRUE)
+
+	return TRUE
```

Comment:
Does this work properly with empty hand clicking? Currently you can open airlocks by clicking on them but I'm wondering if this code overwrites that behavior.

Reply:
fixed. I can open and close airlocks. I can use a crowbar to open the airlocks when unpowered and close them with my hand.